### ISSUE TITLE
My bots keep dying on Lord Seis  +  cta usage

### ISSUE THREAD

**Issue by robcsy9363** [kolton/d2bot-with-kolbot#1215]
I set up a 3 man group for Diablo - Baal runs. Hammerdin, blizz soso and a zeal pala (weird team but whatever...)
My problem is, when they reach Lord de Seis, 5 times out of 10 the closest member of the group running around and getting killed by that lethal fanaticism affected lords, and as a chicken they leave, or even die. (They geared well anyway)
Is there any way to precast some blizzard or some hammers before the leader open the seal, or make them approach more carefully?
If I take control on any of them there is no problem because of the awareness.

About the Cta
How can I make the followers gather up to the leader every time when using cta?
Again they running around like crazy, and very often they do not receive any bo-s.

**Comment by DarkHorseDre** [comment]
short answer is yes, but would require some work to cast at the spot before seal is opened by leader.. someone else may be able to help.

Approaching more carefully would require changes to the scripts.  there are a few issues with how the bot moves/attacks, positions.

Also consider that the range for the attack determines how close they get to the enemies.  hammers (and zeal) for example, are coded to stand real close to the target, orb too (although I would stand farther back for safety and cast more to kill the mob).


**for BO**:

in attack.js I make the bot delay(2000) after opening tp, and i changed precast.js to force the BO, so the follower casts bo when it jumps through tp with other follower and leader. a bit of a hack but works.

in your case youd want leader to precast after casting tp, maybe 0.5-1 second after, as the followers enter tp.
if you do this before it says(kill blah) then the followers should stand there and take bo without moving.

There are better ways of doing it but require more customisations.

**Comment by master2g** [comment]
I noticed that once I put enigma on characters that didn't have it, things worked out 10x more smoothly alot of the time.  I know that's not really a fix, but if you compare a lite fury / charged strike javazon with and without enigma, it is night and day.  Without enigma she pretty much just runs in circles most of the time and shows up late to every battle.

**Comment by DarkHorseDre** [comment]
yeah its cos the scripts position & attack differently with teleport than with melee (also avoids attacks etc by teleing away etc).  the melee/waling stuff doesn't play very well at all.  even the pala code in cs is built for eni. if you run it without eni it will try to tank more and die more lol
to me the scripts were written with an assumed eni. 

for walking you will need to change positing and make it less direct/a bit smarter!
